packages/babel-generator/src/buffer.js

```diff
@@ -14,6 +14,7 @@ export default class Buffer {
     this._indent = format.indent.base;
     this.format = format;
     this.buf = "";
+    this.last = "";
```

Comment:
Can we add a code comment here in case someone ever has this question in the future? It would suck if someone removed this optimization unwittingly.